app/views/publishers/vacancies/job_applications/reference_requests/show.html.slim

```diff
@@ -83,3 +83,23 @@
           - timeline.with_item(key: t(".reference_email_changed", email: version.changeset["email"].last), value: value)
         - elsif version.changeset.key? "marked_as_complete"
           - timeline.with_item(key: t(".marked_as_complete"), value: value)
+
+    .govuk-summary-card
+      .govuk-summary-card__title-wrapper
+        h2.govuk-summary-card__title = t("publishers.vacancies.job_applications.show.notes.heading")
+
+      .govuk-summary-card__content
+        - @job_application.notes.each do |note|
+          p.govuk-body-m = note.content
+          p.govuk-body-s By #{note.publisher.given_name} #{note.publisher.family_name}, #{note.created_at}
+          = govuk_link_to(t("buttons.delete"), organisation_job_job_application_note_path(vacancy.id, @job_application, note, return_to: "reference_request", reference_request_id: params[:id]), method: :delete)
+          hr.govuk-section-break.govuk-section-break--l.govuk-section-break--visible
+
+        = form_for @notes_form, url: organisation_job_job_application_notes_path(vacancy.id, @job_application), html: { data: { "show-hidden-content-target": "content" } } do |f|
+          = f.govuk_error_summary
+          = hidden_field_tag :return_to, "reference_request"
+          = hidden_field_tag :reference_request_id, params[:id]
```

Comment:
Suggestion: 
I suspect we could simplify the return logic if we had a single `hidden_field_tag` with a whole return URL generated here?

EG here:
```
= f.hidden_field_tag :return_to, "organisation_job_job_application_reference_request_path(vacancy.id, @job_application.id, params[:id])"
```

If I am thinking this right, this would make extracting the notes as a component even easier, since we could directly inject the `return_to` URL (as we would inject the `job_application`  as a component parameter, and now all the 3 forms would share the same fields.